### DISCUSSION TITLE
Added to the Lazyman Neovim Configuration Manager

### DISCUSSION THREAD

**Discussion by doctorfree** [jdhao/nvim-config#242] [General]
This configuration has been added to the Personals category of supported Neovim configurations in the [Lazyman Neovim Configuration Manager](https://lazyman.dev).

Lazyman can be used to install, initialize, explore, and manage multiple Neovim configurations. To install and initialize this configuration with Lazyman, after installing Lazyman run the command `lazyman -w Jdhao`.

I've used your configuration in the past to augment my own. It's an excellent configuration to explore, learn, and cobble snippets from. Thanks!

If you have any questions, comments, suggestions, or concerns about the inclusion of this configuration in Lazyman, please let me know or open an issue. Nice work!

**Comment by jdhao** [comment]
hey, thanks for adding my config.
